doc/config.example.yaml

```diff
@@ -368,6 +369,26 @@ registry:
 
   # Zone responsible for flash nodes registration
   flash:
-    # Having separate authoritative server increases redundancy and reduces load,
+    # Having one or more secondary servers increase redundancy and reduces load,
     # hence this configuration, while unlikely in a small to medium network, is supported.
+    authoritative: true
+
+    # Redirect server for REST endpoints of the registry
+    redirect_primary: http://nsHidden.bosagora.io
```

Comment:
Technically we don't have to redirect to a primary, so the name is a bit of a misnomer. But let's leave it as is for now.

Reply:
Renamed this and `primary_servers` and better documented